IBPSA/ThermalZones/ReducedOrder/SolarGain/CorrectionGDoublePane.mo

```diff
@@ -42,14 +42,16 @@ protected
       XN2_diff "Energetic dregree of transmission for second pane";
   parameter Modelica.Units.SI.Emissivity a1_diff=1 - Ta1_diff - rho_1_diff
     "Degree of absorption for single pane window";
-  parameter Modelica.Units.SI.CoefficientOfHeatTransfer Q21_diff=a1_diff*(1 + (
-      Ta1_diff*rho_1_diff/XN2_diff))*UWin/25
-    "Coefficient of heat transfer for exterior pane of double pane window";
-  parameter Modelica.Units.SI.CoefficientOfHeatTransfer Q22_diff=a1_diff*(
-      Ta1_diff/XN2_diff)*(1 - (UWin/7.7))
-    "Coefficient of heat transfer for interior pane of double pane window";
-  parameter Modelica.Units.SI.CoefficientOfHeatTransfer Qsek2_diff=Q21_diff +
-      Q22_diff "Overall coefficient of heat transfer for double pane window";
+  parameter Modelica.Units.SI.CoefficientOfHeatTransfer UWin_dimless=1
+    "Auxiliary parameter for dimensionless UWin";
+  parameter Real Q21_diff=a1_diff*(1 + (
+      Ta1_diff*rho_1_diff/XN2_diff))*UWin/UWin_dimless/25
+    "Auxiliary parameter for heat transfer of exterior pane of double pane window";
+  parameter Real Q22_diff=a1_diff*(
+      Ta1_diff/XN2_diff)*(1 - (UWin/UWin_dimless/7.7))
+    "Auxiliary parameter for heat transfer of interior pane of double pane window";
+  parameter Real Qsek2_diff=Q21_diff + Q22_diff
+    "Overall auxiliary parameter for heat transfer of double pane window";
```

Comment:
Checking `IBPSA.ThermalZones.ReducedOrder.Examples.SimpleRoomFourElements` in OMEdit stil gives the warning
```
[1] 05:50:36 Translation Warning
[Modelica.Blocks.Math: 552:5-552:12]: [The following equation is INCONSISTENT due to specified unit information: thermalZoneFourElements.eRadSol[1].y = thermalZoneFourElements.eRadSol[1].k * thermalZoneFourElements.eRadSol[1].u](omeditmessagesbrowser:///Modelica.Blocks.Math?lineNumber=552)

[2] 05:50:36 Translation Warning
The units of following sub-expressions need to be equal:
- sub-expression "thermalZoneFourElements.eRadSol[1].k * thermalZoneFourElements.eRadSol[1].u" has unit "W/m2"
- sub-expression "thermalZoneFourElements.eRadSol[1].y" has unit "W"
```
Also, once you declare `Q*_diff` as `Real`, I don't think you need `UWin_dimless`, do you? If you indeed need it, make it `final` since then, a tool knows it cannot be changed and the symbolic process can remove it rather than keeping a multiplication by 1.

Reply:
UWin_dimless is not necessary, you are right, thank you.
I actually don't get the new warning in my OMEdit (v.1.19.2), but the problem should be due to a missing unit specification of two gain factors k.